### PR TITLE
BaSP-MR-49: Activities CRUD

### PR DESCRIPTION
**Pull request by francoax**
Development done:
- Display of activities list. 
- Create, edit and delete of activities.

### CODE REVIEW

File: src/Components/Activities/Modals/Delete/deleteModal.module.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
+.modal {
+  position: absolute;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgb(0, 0, 0);
```
Comment (MauroCaffesse) [inline src/Components/Activities/Modals/Delete/deleteModal.module.css]: To maintain consistency in the code, you can use one color format in the styles.

File: src/Components/Activities/Table/List.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import { useState, useEffect } from 'react';
```
Comment (MauroCaffesse) [inline src/Components/Activities/Table/List.jsx:2]:
```suggestion
import React, { useState, useEffect } from 'react';
```
You should import React, like this for example

File: src/Components/Activities/index.jsx [hunk 3]
```diff
@@ -0,0 +1,174 @@
+import { useState, useEffect } from 'react';
```
Comment (MauroCaffesse) [inline src/Components/Activities/index.jsx:2]:
```suggestion
import React, { useState, useEffect } from 'react';
```
Also here

**Review by MauroCaffesse — CHANGES_REQUESTED** [review]

**Review by MauroCaffesse — APPROVED** [review]
Good job!

**Review by BareaMartin — APPROVED** [review]

**Pull request merged**